jekyll/_cci2/insights-tests.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: classic-docs
+title: "Test Insights"
+short-title: "Using Test Insights"
+description: "Using Test Insights"
+version:
+- Cloud
+---
+
+## Overview
+
+Using CircleCI Test Insights, users are able to analyze their test performance
+within their most recent executions and identify opportunities for test
+optimization. To access Test Insights, select the **Tests** tab on the
+**Workflow Insights** page. 
+
+The following test insights are available:
+
+### Performance Summary
+
+
+![]({{ site.baseurl }}/assets/img/docs/insights-test-performance.png)
+
+The summary chart displays test performance across the most recent 10 workflow
+runs. Users can hover over each bar to review the test count, skipped tests, and
+success rate of their tests.
+
+### Top 10 Most Failed Tests 
+
+![]({{ site.baseurl }}/assets/img/docs/insights-test-most-failed.png)
+
+Users can view the 10 tests with the lowest success rates in their most recent
+pipeline executions. The table provides the Test name, the associated Job, run
+time, and success rate.
+
+
+### Top 10 Slowest Tests 
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 29. You can fix by inserting `{: #top-10-slowest-tests }` to the next line